lib/npg_qc/autoqc/checks/pulldown_metrics.pm

```diff
@@ -56,28 +57,51 @@ has 'max_java_heap_size' => (
     default => $MAX_JAVA_HEAP_SIZE,
 );
 
-has 'picard_jar_path' => (
+has 'picard_module' => (
     is      => 'ro',
-    isa     => 'NpgCommonResolvedPathJarFile',
-    coerce  => 1,
-    default => $PICARD_JAR_NAME,
+    isa     => 'Str',
+    default => $PICARD_COMMAND,
 );
 
-has 'picard_command' => (
-    is         => 'ro',
-    isa        => 'Str',
+# TODO: This needs to be factored out of here, insert_size and alignment_filter_metrics
```

Comment:
This comment is incorrect, please remove. Different qc checks have slightly different logic.